virttest/utils_windows/virtio_win.py

```diff
@@ -132,26 +132,33 @@ def _get_netkvmco_path(session):
         raise exceptions.TestError(err)
 
     middle_path = "%s\\%s" % (guest_name, guest_arch)
-    find_cmd = 'dir /b /s %s\\netkvmco.dll | findstr "\\%s\\\\"'
-    find_cmd %= (viowin_ltr,  middle_path)
-    netkvmco_path = session.cmd(find_cmd).strip()
-    LOG.info("Found netkvmco.dll file at %s" % netkvmco_path)
-    return netkvmco_path
-
+    for file_type in ["netkvmco.dll", "netkvmco.exe"]:
+        find_cmd = 'dir /b /s "%s" | findstr "%s" | findstr "%s"' \
+                               % (viowin_ltr, middle_path, file_type)
```

Comment:
Is possible to find `netkvmco` prefix and return the file name? The list is file names but you designed it with `file_type in`.

Reply:
Hi @PaulYuuu, this suggestion still needs time and the other suggestions have been modified and tested. : )

Reply:
Hi @PaulYuuu, This suggestion has been resolved in the latest commit. Could you please check it again?

Reply:
`file_type` --> `file_name`

Reply:
So I don't know more about the `findstr` command, I will accept your current way.

Reply:
> `file_type` --> `file_name`

The typo has been corrected.